.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
You should delete this file. This happens when you are running more than one live-server

Reply:
Deleted this file. Problem Solved.